packages/openassistants/openassistants/core/assistant.py

```diff
@@ -106,12 +106,13 @@ async def do_infilling(
         entities_info: Dict[str, List[Entity]],
     ) -> Tuple[bool, dict]:
         # Perform infilling and generate argument decisions in parallel
+        chat_history: List[OpasMessage] = dependencies.get("chat_history")  # type: ignore
```

Comment:
Not sure what the correct way of handling this is from a typing perspective, for now I think its cleanest to ignore the cast from Any -> List[OpasMessage].  Cleaner to do that as a stand alone statement than bundled into the arguments to generate_arguments.

Reply:
I think this is fine for now, I suspect we will refactor `dependencies` later on as a it's a bit messy now